core/chain_indexer.go

```diff
@@ -775,11 +775,7 @@ func (c *ChainIndexer) addOutpointsToIndexer(addressOutpointsWithBlockHeight map
 			// After the BigSporkFork the minimum conversion period changes to 7200 blocks
 			var lockup *big.Int
 			if lockupByte == 0 {
```

Comment:
Not an error, but technically unnecessary, should just be lockup = new(big.Int).SetUint64(params.LockupByteToBlockDepth[lockupByte])